README.md

```diff
@@ -20,6 +20,16 @@ downloadable for older versions of macOS and iOS on the Mac App Store.
 
 Kiwix developers usually work with latest macOS and Xcode. Check our [Continuous Integration Workflow](https://github.com/kiwix/apple/blob/main/.github/workflows/ci.yml) to find out which XCode version we use on Github Actions.
 
+After checkout please run:
```

Comment:
I thought we wanted to ease onboarding (cf. the move to a swift package) ; this goes the other way.
I'd be reading the README so I'm OK with this but is @kelson42? Because we discussed otherwise:

- having the xcproj in the repo
- having it in .gitignore (so it's not pushed accidentaly)
- in the CI (and/or pre-commit), if `project.yml` is modified, ensure the project is added (with --force) and pushed otherwise fail.

Pros:

- does not require any initial setup from developers
- prevents accidental un-sync

Cons:

- no `git diff Kiwix.xcproject` (but that's the same with what's proposed here ; consequence of xcodegen)
- potential frustration of newcomer not reading readme and wanting to push change to xcproj before realizing it's ignored
- Initial CI failures in PRs when modifying the template and not force-pushing the gen'd xcproj

Reply:
I don't have a strong opinion on this one.
Now I've added back the project files generated by Xcode-gen, maybe as a step 2 we can try to figure out how to keep it in sync.
Any swift file / asset file added to the repo would change the project file, therefore it needs to be re-generated.
Maybe we could apply special GitHub rules who can commit to that file, so on a successful build it will be updated from the CI side, and any change pushed to it will be ignored / overwritten.

Reply:
> I thought we wanted to ease onboarding (cf. the move to a swift package) ; this goes the other way.
> I'd be reading the README so I'm OK with this but is @kelson42?

I confirm